lib/src/glass_container.dart

```diff
@@ -403,7 +403,7 @@ class GlassContainer extends StatelessWidget {
       current = ClipOval(child: current);
     } else {
       assert(borderRadius != null);
-      current = ClipRRect(borderRadius: borderRadius, child: current);
+      current = ClipRRect(borderRadius: borderRadius!, child: current);
```

Comment:
let's use conditional/null-aware(`??`) operators instead of null-assertion(`!`)

Reply:
I did not want to remove the assertion and therefore a conditional operator was not necessary. As far as the `assert` should be removed, you can of course use the `?? BorderRadius.zero`.